api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/EriDSDDataset.java

```diff
@@ -383,27 +383,6 @@ public DataSetDefinition constructEriDSDDataset() {
                     mappings)),
             mappings),
         getChildrenColumn());
-    dsd.addColumn(
-        "N3UBNP",
-        "N3 Patients who are breastfeeding excluding pregnant patients",
-        EptsReportUtils.map(
-            eptsGeneralIndicator.getIndicator(
-                "N3UBNP",
-                EptsReportUtils.map(
-                    eriDSDCohortQueries.getPatientsWhoAreBreastfeedingAndNotPregnantN3(),
-                    mappings)),
-            mappings),
-        "");
-    dsd.addColumn(
-        "N3UPB",
-        "N3: Pregnant: includes breastfeeding patients",
-        EptsReportUtils.map(
-            eptsGeneralIndicator.getIndicator(
-                "N3UPB",
-                EptsReportUtils.map(
-                    eriDSDCohortQueries.getPatientsWhoArePregnantAndBreastfeedingN3(), mappings)),
```

Comment:
@ynurmahomed I think we can remove the cohort `getPatientsWhoArePregnantAndBreastfeedingN3` since it isn't being used anywhere else, same comment for the breastfeeding cohort.

Reply:
@nthfloor removed.